data/ltp/net.sh

```diff
@@ -1,25 +1,30 @@
 #!/bin/sh
 # emulate $LTPROOT/testscripts/network.sh
 
-export TCID="network_settings"
+load_tst_net()
+{
+	export TCID="network_settings"
```

Comment:
what the purpose of this variable ?  you set it here , then copy same value to another variable , then unset it ...
or you using it inside tst_net.sh / test_net.sh ?